app/helpers/tasklist_helper.rb

```diff
@@ -0,0 +1,56 @@
+module TasklistHelper
+  STATUSES = {
+    :not_started => "Not started",
+    :cannot_start_yet =>  "Cannot start yet",
+    :completed => "Completed",
+    :in_progress => "In progress"
+  }
+
+  STYLES = {
+    :not_started => "govuk-tag--grey",
+    :cannot_start_yet => "govuk-tag--grey",
+    :completed => "",
+    :in_progress => "govuk-tag--blue"
+  }
+
+  def get_subsection_status(subsection_name, case_log, questions)
+    if subsection_name == "declaration"
+      return all_questions_completed(case_log) ? :not_started : :cannot_start_yet
+    end
+
+    return :not_started if questions.all? {|question| case_log[question].blank?}
+    return :completed if questions.all? {|question| case_log[question].present?}
+    :in_progress
+  end
+
+  def get_status_style(status_label)
+    STYLES[status_label]
+  end
+
+  def get_status_label(status)
+    STATUSES[status]
+  end
```

Comment:
Can I access these from the module directly?

Reply:
You should be able to do `TasklistHelper::STATUSES`?

Reply:
Oooh, I'll try that!